python/metashape_workflow_functions.py

```diff
@@ -859,16 +863,18 @@ def build_point_cloud(self):
                     crs=Metashape.CoordinateSystem(self.cfg["project_crs"]),
                     subdivide_task=self.cfg["subdivide_task"],
                 )
+                self.written_paths["point_cloud_all_classes"] = output_file #export
             else:
                 # call with classes argument
                 self.doc.chunk.exportPointCloud(
                     path=output_file,
                     source_data=Metashape.PointCloudData,
                     format=Metashape.PointCloudFormatLAZ,
                     crs=Metashape.CoordinateSystem(self.cfg["project_crs"]),
-                    clases=self.cfg["buildPointCloud"]["classes"],
+                    classes=self.cfg["buildPointCloud"]["classes"],  # corrected 'clases' to 'classes'
```

Comment:
Nice catch, was this causing an error for you?

Nitpick: I'd probably remove the comment explaining the change. It may be helpful in review, but will not be useful going forward since the issue will be resolved. If you want to explain a choice like this in the future, you could comment on your PR.